frontend/components/propositions/OccurrenceDetail.tsx

```diff
@@ -36,7 +37,10 @@ const OccurrenceDetail: FC<OccurrenceDetailProps> = ({ occurrence }: OccurrenceD
           )
       )}
 
-      <h2 className="text-center my-3" dangerouslySetInnerHTML={{ __html: occurrence.title }} />
+      <h2
+        className="text-center my-3"
+        dangerouslySetInnerHTML={{ __html: titleCase(occurrence.title) }}
+      />
```

Comment:
[Same as with the `div`](https://github.com/ModularHistory/modularhistory/pull/659/files#r702441243), we should be able to replace this with 
```jsx
<h2 className="...">
  titleCase(...)
</h2>
```

Reply:
> [Same as with the `div`](https://github.com/ModularHistory/modularhistory/pull/659/files#r702441243), we should be able to replace this with
> 
> ```js
> <h2 className="...">
>   titleCase(...)
> </h2>
> ```

Got ya. Fixed it and will be pushing the updated code!